plugins/woocommerce/src/StoreApi/Utilities/CheckoutTrait.php

```diff
@@ -85,7 +85,7 @@ private function process_payment( \WP_REST_Request $request, PaymentResult $paym
 				throw new RouteException( 'woocommerce_rest_checkout_invalid_payment_result', __( 'Invalid payment result received from payment method.', 'woocommerce' ), 500 );
 			}
 		} catch ( \Exception $e ) {
-			throw new RouteException( 'woocommerce_rest_checkout_process_payment_error', $e->getMessage(), 402 );
+			throw new RouteException( 'woocommerce_rest_checkout_process_payment_error', $e->getMessage(), 400 );
```

Comment:
We should escape `$e->getMessage()` here. Since the message comes from an unknown source. I'd say `esc_html` should be fine since this is displayed in a notice on the page.

This is a new linting rule, which is why it's being flagged for this line.

Reply:
Thanks @opr ! Weird that it wasn't flagged before - changed in 8e4e15d8c6e3db17a85a2c82ffac9df032102ac0